roles/mirror_ocp_release/tasks/facts.yml

```diff
@@ -7,12 +7,27 @@
   register: release_image
   changed_when: false
 
-- name: "Write rhcos.json from command (>= 4.8)"
+- name: Write rhcos.json from command (4.16+)
+  when: mor_version is version("4.16", ">=")
+  containers.podman.podman_container:
+    name: extract-rhcos
+    image: quay.io/centos/centos:stream9
+    command:
+      - /bin/bash
+      - -c
```

Comment:
From our discussion in Slack, I thought the final fix was using a `|` after the `-c`?
```
          - -c
          - |
            /ocp/openshift-baremetal-install coreos print-stream-json > /ocp/rhcos-test.json
```

Reply:
Ah, good observation, I thought about it as well. But I realized that the truly important part is to have the command *and* the redirect in the same line, I tested without the pipe and got the expected result.

Reply:
yeah, no pipe is needed. I tested locally and works